src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -314,7 +314,7 @@ private async Task ApplyContext(HostAssignmentContext assignmentContext)
             var options = _optionsFactory.Create(ScriptApplicationHostOptionsSetup.SkipPlaceholder);
             RunFromPackageContext pkgContext = assignmentContext.GetRunFromPkgContext();
 
-            if (_environment.SupportsAzureFileShareMount())
+            if (_environment.SupportsAzureFileShareMount(pkgContext.Url))
```

Comment:
we should be able to make it work without passing anything. please check comments on the method definition

Reply:
I am not passing `pkgContext.Url` anymore